gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py

```diff
@@ -56,6 +56,57 @@ class FakeObjects(object):
                               '"gateway_ip": "1.2.3.4"}]')
     data_for_del_src_route = '[{"source_cidr": "1.2.3.4/24"}]'
     timeout = 120
+    content_headers = {'Content-Type': 'application/json'}
+    asav_url = 'https://172.24.4.5'
+    asav_bulk_cli_url = '%s/api/cli' % asav_url
+    asav_api_cli_url = '%s/api' % asav_url
+
+    def fake_asav_resources(self, resource):
```

Comment:
We should keep all the asav related test data in a different file. For submission, there should not be asav test code in unit tests that we submit.